recipe/scripts/activate.sh

```diff
@@ -8,10 +8,15 @@ if [ -n "$PROJ_LIB" ]; then
 fi
 
 
-if [ -d ${CONDA_PREFIX}/share/proj ]; then
-  export PROJ_LIB=${CONDA_PREFIX}/share/proj
-elif [ -d ${CONDA_PREFIX}/Library/share/proj ]; then
-  export PROJ_LIB=${CONDA_PREFIX}/Library/share/proj
+if [ -d "${CONDA_PREFIX}/share/proj" ]; then
+  export "PROJ_LIB=${CONDA_PREFIX}/share/proj"
+elif [ -d "${CONDA_PREFIX}/Library/share/proj" ]; then
+  export PROJ_LIB="${CONDA_PREFIX}/Library/share/proj"
 fi
 
-export PROJ_NETWORK="ON"
+if [ -f "${CONDA_PREFIX}/share/proj/copyright_and_licenses.csv" ]; then
+  # proj-data is installed because its license was copied over
+  export PROJ_NETWORK="OFF"
+else
+  export PROJ_NETWORK="ON"
+fi
```

Comment:
@ocefpaf and @snowman2, I realize this isn't the most elegant way to figure out if `proj-data` is installed.  We could create some other file such as `.proj-data` that gets put into this folder to indicate that `proj-data` has been installed.  What do you think.

Reply:
Sounds good. But I'm also OK with what you did here. It is not elegant but "good enough."

Reply:
If it's good enough, I think I'll probably stick with this.